lib/RxTouch.js

```diff
@@ -1,8 +1,32 @@
 const s3 = require("../lib/s3");
 
-module.exports.noAppointmentMatchString = "no locations with available appointments";
+function ScheduleUrl(siteName, zip) {
+    return `https://${siteName.toLowerCase()}sched.rxtouch.com/rbssched/program/covid19/Patient/Schedule?zip=${zip}&appointmentType=5954`;
+}
 
-module.exports.ScrapeRxTouch = async function ScrapeRxTouch(browser, site, siteName) {
+function CalendarUrl(siteName) {
+    return `https://${siteName.toLowerCase()}sched.rxtouch.com/rbssched/program/covid19/Calendar/PatientCalendar`;
+}
+
+function CalendarDayUrl(siteName) {
+    return `https://${siteName.toLowerCase()}sched.rxtouch.com/rbssched/program/covid19/Calendar/PatientCalendarDay`;
+}
```

Comment:
I don't actually need these as functions. I'll move this down into the body of the main function (they're only used once each).